src/clients/bridges/ArbitrumAdapter.ts

```diff
@@ -67,18 +67,7 @@ export class ArbitrumAdapter extends CCTPAdapter {
     readonly spokePoolClients: { [chainId: number]: SpokePoolClient },
     monitoredAddresses: string[]
   ) {
-    super(spokePoolClients, 42161, monitoredAddresses, logger, [
-      "USDC",
-      "USDT",
-      "WETH",
-      "DAI",
-      "WBTC",
-      "UMA",
-      "BADGER",
-      "BAL",
-      "ACX",
-      "POOL",
-    ]);
+    super(spokePoolClients, 42161, monitoredAddresses, logger, SUPPORTED_TOKENS[CHAIN_IDs.ARBITRUM]);
```

Comment:
It's also tempting to suggest this, but I guess most of this code will go away shortly anyway so it might be redundant.

```suggestion
    const { ARBITRUM } =  CHAIN_IDs;
    super(spokePoolClients, ARBITRUM, monitoredAddresses, logger, SUPPORTED_TOKENS[ARBITRUM]);
```

Reply:
Yeah it probably does not hurt to do this, so I made the change here: https://github.com/across-protocol/relayer/pull/1618/commits/472dc40e9b31b780f308733266d170ac811135d3